android build FAILURE:

## Before I have submitted the issue 
[ ] I have read an [installation guide](https://tipsi.github.io/tipsi-stripe/docs/installation.html)  
[ ] I know that for an iOS I need to install pods because I've read the [installation guide](https://tipsi.github.io/tipsi-stripe/docs/installation.html#cocoapods-ios)  
[ ] I have read a [linking guide](https://tipsi.github.io/tipsi-stripe/docs/linking.html) and checked that everything is OK like in [manual linking guide](https://tipsi.github.io/tipsi-stripe/docs/linking.html#manual)  
[ ] I know that before using `tipsi-stripe` I need to set options for my app as described in [usage guide](https://tipsi.github.io/tipsi-stripe/docs/usage.html)  

## The problem
> Task :tipsi-stripe:compileDebugJavaWithJavac FAILED
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/RedirectUriReceiver.java:5: error: packa
ge android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:3: error: package a
ndroid.support.annotation does not exist
import android.support.annotation.NonNull;
                                 ^
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:4: error: package android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:5: error: package android.support.annotation does not exist
import android.support.annotation.NonNull;
                                 ^
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/Errors.java:3: error: package android.support.annotation does not exist
import android.support.annotation.NonNull;
                                 ^
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/PayFlow.java:5: error: package android.support.annotation does not exist
import android.support.annotation.NonNull;
                                 ^
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/StripeModule.java:7: error: package android.support.annotation does not exist
import android.support.annotation.NonNull;
                                 ^
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/StripeModule.java:8: error: package android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/dialog/AddCardDialogFragment.java:8: error: package android.support.v4.content does not exist
import android.support.v4.content.ContextCompat;
                                 ^
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/OpenBrowserActivity.java:7: error: package android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/RedirectUriReceiver.java:14: error: cannot find symbol
  protected void onCreate(@Nullable Bundle savedInstanceState) {
                           ^
  symbol:   class Nullable
  location: class RedirectUriReceiver
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:209: error: cannot find symbol
  public static WritableMap convertSourceToWritableMap(@Nullable Source source) {
                                                        ^
  symbol:   class Nullable
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:208: error: cannot find symbol
  @NonNull
   ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:237: error: cannot find symbol
  public static WritableMap stringMapToWritableMap(@Nullable Map<String, String> map) {
                                                    ^
  symbol:   class Nullable
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:236: error: cannot find symbol
  @NonNull
   ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:252: error: cannot find symbol
  public static WritableMap convertOwnerToWritableMap(@Nullable final SourceOwner owner) {
                                                       ^
  symbol:   class Nullable
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:251: error: cannot find symbol
  @NonNull
   ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:272: error: cannot find symbol
  public static WritableMap convertAddressToWritableMap(@Nullable final Address address) {
                                                         ^
  symbol:   class Nullable
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:271: error: cannot find symbol
  @NonNull
   ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:290: error: cannot find symbol
  public static WritableMap convertReceiverToWritableMap(@Nullable final SourceReceiver receiver) {
                                                          ^
  symbol:   class Nullable
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:289: error: cannot find symbol
  @NonNull
   ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:306: error: cannot find symbol
  public static WritableMap convertRedirectToWritableMap(@Nullable SourceRedirect redirect) {
                                                          ^
  symbol:   class Nullable
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:305: error: cannot find symbol
  @NonNull
   ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:321: error: cannot find symbol
  public static WritableMap convertCodeVerificationToWritableMap(@Nullable SourceCodeVerification codeVerification) {
                                                                  ^
  symbol:   class Nullable
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:320: error: cannot find symbol
  @NonNull
   ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:335: error: cannot find symbol
  public static WritableMap mapToWritableMap(@Nullable Map<String, Object> map){
                                              ^
  symbol:   class Nullable
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:334: error: cannot find symbol
  @NonNull
   ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:349: error: cannot find symbol
  public static void pushRightTypeToMap(@NonNull WritableMap map, @NonNull String key, @NonNull Object object) {
                                         ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:349: error: cannot find symbol
  public static void pushRightTypeToMap(@NonNull WritableMap map, @NonNull String key, @NonNull Object object) {
                                                                   ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:349: error: cannot find symbol
  public static void pushRightTypeToMap(@NonNull WritableMap map, @NonNull String key, @NonNull Object object) {
                                                                                        ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:406: error: cannot find symbol
  public static String getStringOrNull(@NonNull ReadableMap map, @NonNull String key) {
                                        ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/util/Converters.java:406: error: cannot find symbol
  public static String getStringOrNull(@NonNull ReadableMap map, @NonNull String key) {
                                                                  ^
  symbol:   class NonNull
  location: class Converters
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:57: error: cannot find symbol
  public GoogleApiPayFlowImpl(@NonNull Fun0<Activity> activityProvider) {
                               ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:61: error: cannot find symbol
  private PaymentsClient createPaymentsClient(@NonNull Activity activity) {
                                               ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:67: error: cannot find symbol
  private void isReadyToPay(@NonNull Activity activity, boolean isExistingPaymentMethodRequired, @NonNull final Promise promise) {
                             ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:67: error: cannot find symbol
  private void isReadyToPay(@NonNull Activity activity, boolean isExistingPaymentMethodRequired, @NonNull final Promise promise) {
                                                                                                  ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:121: error: cannot find symbol
  private PaymentDataRequest createPaymentDataRequest(@NonNull final String totalPrice,
                                                       ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:122: error: cannot find symbol
                                                      @NonNull final String currencyCode,
                                                       ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:127: error: cannot find symbol
                                                      @NonNull final Collection<String> countryCodes
                                                       ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:169: error: cannot find symbol
  private void startPaymentRequest(@NonNull Activity activity, @NonNull PaymentDataRequest request) {
                                    ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:169: error: cannot find symbol
  private void startPaymentRequest(@NonNull Activity activity, @NonNull PaymentDataRequest request) {
                                                                ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:182: error: cannot find symbol
  public void paymentRequestWithAndroidPay(@NonNull ReadableMap payParams, @NonNull Promise promise) {
                                            ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:182: error: cannot find symbol
  public void paymentRequestWithAndroidPay(@NonNull ReadableMap payParams, @NonNull Promise promise) {
                                                                            ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java:200: error: cannot find symbol
  public void deviceSupportsAndroidPay(boolean isExistingPaymentMethodRequired, @NonNull Promise promise) {
                                                                                 ^
  symbol:   class NonNull
  location: class GoogleApiPayFlowImpl
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/Errors.java:30: error: cannot find symbol
  static String toErrorCode(@NonNull Exception exception) {
                             ^
  symbol:   class NonNull
  location: class Errors
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/Errors.java:39: error: cannot find symbol
  static String getErrorCode(@NonNull ReadableMap errorCodes, @NonNull String errorKey) {
                              ^
  symbol:   class NonNull
  location: class Errors
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/Errors.java:39: error: cannot find symbol
  static String getErrorCode(@NonNull ReadableMap errorCodes, @NonNull String errorKey) {
                                                               ^
  symbol:   class NonNull
  location: class Errors
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/Errors.java:43: error: cannot find symbol
  static String getDescription(@NonNull ReadableMap errorCodes, @NonNull String errorKey) {
                                ^
  symbol:   class NonNull
  location: class Errors
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/Errors.java:43: error: cannot find symbol
  static String getDescription(@NonNull ReadableMap errorCodes, @NonNull String errorKey) {
                                                                 ^
  symbol:   class NonNull
  location: class Errors
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/PayFlow.java:18: error: cannot find symbol
  protected final @NonNull Fun0<Activity> activityProvider;
                   ^
  symbol:   class NonNull
  location: class PayFlow
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/PayFlow.java:23: error: cannot find symbol
  public PayFlow(@NonNull Fun0<Activity> activityProvider) {
                  ^
  symbol:   class NonNull
  location: class PayFlow
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/PayFlow.java:59: error: cannot find symbol
  public void setPublishableKey(@NonNull String publishableKey) {
                                 ^
  symbol:   class NonNull
  location: class PayFlow
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/PayFlow.java:87: error: cannot find symbol
  public static boolean isPlayServicesAvailable(@NonNull Activity activity) {
                                                 ^
  symbol:   class NonNull
  location: class PayFlow
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/StripeModule.java:55: error: cannot find symbol
  @Nullable
   ^
  symbol:   class Nullable
  location: class StripeModule
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/StripeModule.java:58: error: cannot find symbol
  @Nullable
   ^
  symbol:   class Nullable
  location: class StripeModule
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/StripeModule.java:93: error: cannot find symbol
  public void init(@NonNull ReadableMap options, @NonNull ReadableMap errorCodes) {
                    ^
  symbol:   class NonNull
  location: class StripeModule
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/StripeModule.java:93: error: cannot find symbol
  public void init(@NonNull ReadableMap options, @NonNull ReadableMap errorCodes) {
                                                  ^
  symbol:   class NonNull
  location: class StripeModule
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/StripeModule.java:131: error: cannot find symbol
  private static int androidPayModeToEnvironment(@NonNull String androidPayMode) {
                                                  ^
  symbol:   class NonNull
  location: class StripeModule
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/StripeModule.java:315: error: cannot find symbol
  void processRedirect(@Nullable Uri redirectData) {
                        ^
  symbol:   class Nullable
  location: class StripeModule
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/OpenBrowserActivity.java:19: error: cannot find symbol
  protected void onCreate(@Nullable Bundle savedInstanceState) {
                           ^
  symbol:   class Nullable
  location: class OpenBrowserActivity
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/dialog/AddCardDialogFragment.java:123: error: cannot find symbol
    doneButton.setTextColor(ContextCompat.getColor(getActivity(), R.color.colorAccent));
                            ^
  symbol:   variable ContextCompat
  location: class AddCardDialogFragment
/Users/user/Projekts/dryft/mobile/node_modules/tipsi-stripe/android/src/main/java/com/gettipsi/stripe/dialog/AddCardDialogFragment.java:124: error: cannot find symbol
    dialog.getButton(AlertDialog.BUTTON_NEGATIVE).setTextColor(ContextCompat.getColor(getActivity(), R.color.colorAccent));
                                                               ^
  symbol:   variable ContextCompat
  location: class AddCardDialogFragment
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
62 errors

## Environment
* `tipsi-stripe` version: 
* Last `tipsi-stripe` version where the issue was not reproduced (if applicable): 
* iOS or Android: 
* OS version: 
* React-Native version: 
* (Android only) `com.google.firebase:firebase-core` version: 
* (Android only) `com.google.android.gms:play-services-base` version: 

## Links to logs and sources
Create a [GIST](https://gist.github.com) which is a paste of your _full_ logs or sources, and link them here. 
If you are reporting a bug, _always_ include build or error logs!

* [ios/Podfile](https://gist.github.com/link_to_podfile)
* [android/build.gradle](https://gist.github.com/link_to_podfile)

For `Android`, please provide the following sections from `android/app/build.gradle`:
* android.compileSdkVersion
* android.buildToolsVersion
* android.defaultConfig.minSdkVersion
* android.defaultConfig.targetSdkVersion
* android.defaultConfig.multiDexEnabled (if exists)

## Screenshots, GIFs (Must to have)
Just drag-and-drop them to this textarea

## Code To Reproduce Issue (Good To Have)
Please remember that with sample code it's easier to reproduce the bug and it's much faster to fix it.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -35,7 +35,7 @@ dependencies {
   implementation fileTree(include: ['*.jar'], dir: 'libs')
   implementation 'com.facebook.react:react-native:+'
   implementation 'androidx.legacy:legacy-support-v4:1.0.0'
-  implementation 'com.androidx.appcompat:1.1.0'
+  implementation 'androidx.appcompat:appcompat:1.1.0'
   implementation "com.google.android.gms:play-services-wallet:$googlePlayServicesVersion"
   implementation "com.google.firebase:firebase-core:$firebaseVersion"
   implementation 'com.stripe:stripe-android:8.1.0'

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -15,7 +15,7 @@ android {
     targetSdkVersion rootProject.hasProperty('targetSdkVersion') ? rootProject.targetSdkVersion : DEFAULT_TARGET_SDK_VERSION
     versionCode 1
     versionName "1.0"
-    testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     manifestPlaceholders = [
       tipsiStripeRedirectScheme: "example"
     ]
@@ -34,8 +34,8 @@ dependencies {
 
   implementation fileTree(include: ['*.jar'], dir: 'libs')
   implementation 'com.facebook.react:react-native:+'
-  implementation 'com.android.support:support-v4:27.1.0'
-  implementation 'com.android.support:appcompat-v7:27.1.0'
+  implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+  implementation 'com.androidx.appcompat:1.1.0'
   implementation "com.google.android.gms:play-services-wallet:$googlePlayServicesVersion"
   implementation "com.google.firebase:firebase-core:$firebaseVersion"
   implementation 'com.stripe:stripe-android:8.1.0'

--- a/android/src/main/java/com/gettipsi/stripe/Errors.java
+++ b/android/src/main/java/com/gettipsi/stripe/Errors.java
@@ -1,6 +1,6 @@
 package com.gettipsi.stripe;
 
-import android.support.annotation.NonNull;
+import androidx.annotation.NonNull;
 
 import com.facebook.react.bridge.ReadableMap;
 import com.gettipsi.stripe.util.ArgCheck;

--- a/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java
+++ b/android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java
@@ -2,7 +2,7 @@ package com.gettipsi.stripe;
 
 import android.app.Activity;
 import android.content.Intent;
-import android.support.annotation.NonNull;
+import androidx.annotation.NonNull;
 
 import com.facebook.react.bridge.Promise;
 import com.facebook.react.bridge.ReadableMap;

--- a/android/src/main/java/com/gettipsi/stripe/OpenBrowserActivity.java
+++ b/android/src/main/java/com/gettipsi/stripe/OpenBrowserActivity.java
@@ -4,7 +4,7 @@ import android.app.Activity;
 import android.content.Intent;
 import android.net.Uri;
 import android.os.Bundle;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
 
 /**
  * Created by remer on 16/11/17.
@@ -22,7 +22,7 @@ public class OpenBrowserActivity extends Activity {
 
     url = getIntent().getStringExtra(EXTRA_URL);
     Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(url))
-        .addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY | Intent.FLAG_ACTIVITY_CLEAR_TOP | 
+        .addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY | Intent.FLAG_ACTIVITY_CLEAR_TOP |
           Intent.FLAG_ACTIVITY_SINGLE_TOP | Intent.FLAG_ACTIVITY_NEW_TASK);
     startActivity(browserIntent);
   }

--- a/android/src/main/java/com/gettipsi/stripe/PayFlow.java
+++ b/android/src/main/java/com/gettipsi/stripe/PayFlow.java
@@ -2,7 +2,7 @@ package com.gettipsi.stripe;
 
 import android.app.Activity;
 import android.content.Intent;
-import android.support.annotation.NonNull;
+import androidx.annotation.NonNull;
 import android.util.Log;
 
 import com.facebook.react.bridge.Promise;

--- a/android/src/main/java/com/gettipsi/stripe/RedirectUriReceiver.java
+++ b/android/src/main/java/com/gettipsi/stripe/RedirectUriReceiver.java
@@ -2,7 +2,7 @@ package com.gettipsi.stripe;
 
 import android.app.Activity;
 import android.os.Bundle;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
 import android.util.Log;
 
 /**

--- a/android/src/main/java/com/gettipsi/stripe/StripeModule.java
+++ b/android/src/main/java/com/gettipsi/stripe/StripeModule.java
@@ -4,8 +4,8 @@ import android.app.Activity;
 import android.content.Intent;
 import android.net.Uri;
 import android.os.AsyncTask;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import android.text.TextUtils;
 
 import com.facebook.react.bridge.ActivityEventListener;

--- a/android/src/main/java/com/gettipsi/stripe/dialog/AddCardDialogFragment.java
+++ b/android/src/main/java/com/gettipsi/stripe/dialog/AddCardDialogFragment.java
@@ -5,7 +5,7 @@ import android.app.Dialog;
 import android.app.DialogFragment;
 import android.content.DialogInterface;
 import android.os.Bundle;
-import android.support.v4.content.ContextCompat;
+import androidx.core.content.ContextCompat;
 import android.text.Editable;
 import android.text.TextWatcher;
 import android.view.View;

--- a/android/src/main/java/com/gettipsi/stripe/util/Converters.java
+++ b/android/src/main/java/com/gettipsi/stripe/util/Converters.java
@@ -1,7 +1,7 @@
 package com.gettipsi.stripe.util;
 
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import android.text.TextUtils;
 
 import com.facebook.react.bridge.Arguments;
@@ -64,11 +64,11 @@ public class Converters {
 
     billingContactMap.putString("emailAddress", emailAddress);
     shippingContactMap.putString("emailAddress", emailAddress);
-    
+
 
     extra.putMap("billingContact", billingContactMap);
     extra.putMap("shippingContact", shippingContactMap);
-    
+
     tokenMap.putMap("extra", extra);
 
     return tokenMap;

--- a/example/android/app/build.gradle
+++ b/example/android/app/build.gradle
@@ -146,7 +146,7 @@ dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "androidx.legacy:legacy-support-v4:1.0.0"
     implementation "com.android.support:design:27.1.0"
-    implementation "com.androidx.appcompat:1.1.0"
+    implementation "androidx.appcompat:appcompat:1.1.0"
     implementation "com.facebook.react:react-native:+"  // From node_modules
 }
 

--- a/example/android/app/build.gradle
+++ b/example/android/app/build.gradle
@@ -144,9 +144,9 @@ android {
 dependencies {
     implementation project(':tipsi-stripe')
     implementation fileTree(dir: "libs", include: ["*.jar"])
-    implementation "com.android.support:support-v4:27.1.0"
+    implementation "androidx.legacy:legacy-support-v4:1.0.0"
     implementation "com.android.support:design:27.1.0"
-    implementation "com.android.support:appcompat-v7:27.1.0"
+    implementation "com.androidx.appcompat:1.1.0"
     implementation "com.facebook.react:react-native:+"  // From node_modules
 }
 

